content/library/api/connections/connections-snowflake.md

```diff
@@ -11,6 +11,43 @@ This page only contains the `st.connections.SnowflakeConnection` class. For a de
 
 <Autofunction function="streamlit.connections.SnowflakeConnection" />
 
+### Configuration
```

Comment:
100% your call @sfc-gh-dmatthews but FWIW I had this at H3 originally and felt like it looked better at H2

![image](https://github.com/streamlit/docs/assets/116604821/eca84d1a-4306-4eea-b6d0-85cc8135ab72)

Reply:
I can be convinced either way. I agree the H2+H3 is prettier than the H3+H4, but logically it's a child to the class which is hard-coded as H2. I mentioned this in Slack, but I want to make some changes to the header sizing and be able to make a class H1 with its methods as H2.

Happy to go either way and address it later for larger consistency.

Reply:
OK. Yeah I think it should eventually be the same level as the methods. Whatever is easier for you.

Reply:
I'm good to merge it like this if you are going to fix the consistency later :)